vllm/model_executor/layers/rotary_embedding.py

```diff
@@ -40,13 +40,20 @@ def __init__(
         max_position_embeddings: int,
         base: int,
         is_neox_style: bool,
+        is_glm_style: bool = False,
```

Comment:
Can you change this to:
```suggestion
        style: str = "neox",
```
and set style to `"neox", "gptj", "glm"` accordingly?